ITEvents/Screens/Search tab/SearchViewController.swift

```diff
@@ -22,11 +22,23 @@ class SearchViewController: UIViewController, ISearchView {
         presenter.setup()
     }
     
+    override func viewWillAppear(_ animated: Bool) {
```

Comment:
If we need to fetch flag from the settings only once (during the initialization of the view) we can move all code that is in ```presenter.updateViewSettings()``` into ```presenter.setup()```.
If so we can remove this ```viewWillAppear``` and also we can remove ```updateViewSettings()``` from ```ISearchPresenter```.

Otherwise if we need to check settings all the time view being shown then I am wrong and forget about it :) 👍

Reply:
We have to do this every time view will appear. If we open search tab, then open settings in another tab, change layout and go to search tab again, viewDidLoad won't be called as view has been already loaded. So we should call this code in viewDidAppear.